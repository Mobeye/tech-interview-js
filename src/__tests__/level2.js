import * as fs from "fs";
import { level2 } from "../answers";

it("level2", () => {
  const data = JSON.parse(fs.readFileSync("./src/level2.json", "utf8"));
  expect(level2(data)).toMatchSnapshot();
});
