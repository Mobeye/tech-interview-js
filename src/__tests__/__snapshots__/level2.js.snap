// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`level2 1`] = `
Object {
  "carts": Array [
    Object {
      "id": 1,
      "total": 2000,
    },
    Object {
      "id": 2,
      "total": 1800,
    },
    Object {
      "id": 3,
      "total": 800,
    },
  ],
}
`;
